C4SGWeb/.gitignore

```diff
@@ -39,3 +39,35 @@ testem.log
 #System Files
 .DS_Store
 Thumbs.db
+
+ehthumbs.db
+Icon?
+*.log
+dist
+build
+
+# Eclipse
+.classpath
+.project
+.settings/
+
+# STS
+.apt_generated
+.factorypath
+.settings
+.springBeans
+
+# IntelliJ IDEA
+.idea/
+*.iws
+*.iml
+*.ipr
+
+# NetBeans
+nbproject/private/
+build/
+nbbuild/
+dist/
+nbdist/
+.nb-gradle/
+/bin/
```

Comment:
The current gitignore file  already has those configurations.
PS: seems your gitignore is out of date !